spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1370,9 +1364,6 @@
           :with_w2s_synced
         )
       }
-      let(:submission) {
-        create :efile_submission, tax_return: nil, data_source: intake
-      }
```

Comment:
are all these removals of `let(:submission) ...` unrelated to the rest of this PR and just general cleanup?

Reply:
Yeah, I realized it was not necessary to have all this repeated code and tried to do some tidying. There's still more that we could do but I went for the lowest hanging fruit